book/scheduling.md

```diff
@@ -1527,8 +1526,8 @@ it was sent and call `set_needs_raster_and_draw` as needed. Because this call
 will come from another thread, we'll need to acquire a lock. Another important
 step is to not clear the `animation_timer` object until *after* the next
 commit occurs. Otherwise multiple rendering tasks could be queued at the same
-time. Finally, rename `scroll` to `active_tab_scroll` and `url` to
-`active_tab_url` to make more clear what they mean.
+time. Finally, save `scroll` in `active_tab_scroll` and `url` in
```

Comment:
The text is describing a rename of existing fields..so it's correct as is?

Reply:
I think it's not—it's describing a copy from the `CommitData` object into brand-new fields on the `Browser` object. Until now the `Browser` hadn't had `url` or `scroll` fields, it'd read them off the `active_tab`.